src/openeo_gfmap/utils/catalogue.py

```diff
@@ -211,26 +215,46 @@ def s1_area_per_orbitstate_vvvh(
     spatial_extent = box(*bounds)
 
     # Computes if there is the full overlap for each of those states
-    union_ascending = unary_union(ascending_products)
-    union_descending = unary_union(descending_products)
+    union_ascending = unary_union(ascending_products[0])
```

Comment:
I figure its only important to check for overlap on the startdate?

Reply:
is there any safeguard in place in case ascending_products is empty and does not have a [0] entry?

Reply:
yes, so here I would put some safeguards in case unary_union does not give a double back. Else gfmap will fail with little explanation why

Reply:
?

Reply:
Okay so I changed the variable capturing the output of `_parse_cdse_products` to two variables `ascending_products` & `ascending_timestamps` and also those are lists so if they are empty the result of the union will just be empty